iam_redpanda_agent.tf

```diff
@@ -222,11 +222,44 @@ data "aws_iam_policy_document" "redpanda_agent1" {
         "CreateLaunchTemplate",
         "RunInstances",
         "CreateVpcEndpointServiceConfiguration",
-        "CreatePlacementGroup"
+        "CreatePlacementGroup",
+        "CreateTags"
       ]
     }
   }
 
+  statement {
```

Comment:
There is a size limit for policies, redpanda_agent1 is already at the limit which is why we have redpanda_agent2. But this may put us over the edge and require creation of redpanda_agent3.